frontend/src/components/notification/notification.tsx

```diff
@@ -0,0 +1,52 @@
+import React, {useEffect} from 'react';
+import {Alert, Snackbar} from "@mui/material";
+import CheckIcon from '@mui/icons-material/Check';
+import {useAppDispatch, useAppSelector} from "../../stores/redux/hooks";
+import {selectNotification, showNotification} from "../../stores/redux/notification/notificationSlice";
+import {INotification} from "../../types/notification";
+import {NotificationType} from "../../types/notificationType";
+
+export const createNotification = (message: string, type: NotificationType): INotification => {
+    return {
+        show: true,
+        type,
+        message
+    };
+};
+
+const Notification: React.FC = () => {
+    const [message, setMessage] = React.useState<string>("");
+    const [show, setShow] = React.useState<boolean>(false);
+    const dispatch = useAppDispatch();
+    const notification = useAppSelector(selectNotification);
+
+    const closeNotification = (event?: React.SyntheticEvent | Event, reason?: string) => {
+        if (reason === 'clickaway') {
+            return;
+        }
+        dispatch(showNotification({show: false}));
+    };
+
+    const updateNotification = () => {
+        if (notification.message) {
+            setMessage(notification.message);
+        }
+        setShow(notification.show);
+    }
+
+    useEffect(() => {
+        updateNotification();
+    }, [notification]);
+
+    return (
+        <Snackbar open={show} autoHideDuration={4000} onClose={closeNotification}
+                  anchorOrigin={{vertical: 'bottom', horizontal: 'center'}}>
+            <Alert variant="filled" icon={<CheckIcon fontSize="inherit"/>} severity={notification.type}
+                   onClose={closeNotification}>
```

Comment:
your icon choice does not much all cases. It  icon represents success, even if the message is a warning or error